client/.eslintrc.json

```diff
@@ -0,0 +1,39 @@
+{
+    "root": true,
+    "env": {
+        "browser": true,
+        "es2021": true,
+        "node": true
+    },
+    "extends": [
+        "plugin:react/recommended",
+        "airbnb"
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true
+        },
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react",
+        "@typescript-eslint"
+    ],
+    "settings": {
+        "import/resolver": {
+            "node": {
+                "extensions": [".js", ".jsx", ".ts", ".tsx"]
+            }
+        }
+    },
+    "rules": {
+        "import/extensions": [1, "ignorePackages", { "ts": "never", "tsx": "never" } ],
+         "react/jsx-filename-extension": ["warn", { "extensions": [".tsx"] }],
+         "@typescript-eslint/no-unused-vars": [
+            "error"
+          ],
```

Comment:
I think these settings is for react typescript,
But we're not using typescript in the client side